crates/astria-cli/src/commands/rollup.rs

```diff
@@ -99,9 +103,24 @@ fn update_yaml_value(
 /// * If the config file cannot be created
 /// * If the arguments cannot be serialized to yaml
 /// * If the yaml cannot be written to the file
-pub(crate) fn create_config(args: &ConfigCreateArgs) -> eyre::Result<()> {
+pub(crate) async fn create_config(args: &ConfigCreateArgs) -> eyre::Result<()> {
     // create rollup from args
-    let rollup = Rollup::try_from(args)?;
+    let mut rollup = Rollup::try_from(args)?;
```

Comment:
why not check `if args.sequencer_initial_block_height.is_some()` instead of the check below? or change `initial_sequencer_height` to be an option?

also update here https://github.com/astriaorg/astria/blob/95864ea139ef5532c0bc3e629f89faea49452e50/crates/astria-cli/src/types.rs#L81